docs/documentation/cert-manager.md

```diff
@@ -91,7 +91,7 @@ spec:
 
 #### Usage
 
-To let the application access the path mounted, we have then to update also the properties of the application (Quarkus, Spring Boot, ...) to use the files that are mounted in the folder "/etc/certs" from the secret.
+To let the application (Quarkus, Spring Boot, ...) access the files mounted under the path "/etc/certs" from the secret, we have then to update also the properties of the application to access them.
```

Comment:
Should be "To let the application (Quarkus, Spring Boot, ...) to access ..."

Reply:
Let is never followed by `to`: https://dictionary.cambridge.org/grammar/british-grammar/let-let-s

Reply:
But `to` here should be added after `application to access`. Ask maybe to Georgios as we are not sure ;-

Reply:
I would write it more idiomatically like this:

```
For an application to be able to access the files mounted under `/etc/certs` from the secret, the application properties must also be updated.
```

Reply:
updated